.github/workflows/test-tutorials.yml

```diff
@@ -51,7 +52,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [18, 20, 22]
+        node-version: [20, 22]
```

Comment:
Why not tested on node v18?

Reply:
I thought node test runner was introduced in 20, but I guess it will work in 18 as well.  Let's find out.